test/functional/tests/conftest.py

```diff
@@ -304,6 +328,9 @@ def __drbd_cleanup():
     if installer.check_if_installed():
         casadm.stop_all_caches()
 
+    remove("/etc/drbd.d/caches.res", force=True, ignore_errors=True)
+    remove("/etc/drbd.d/cores.res", force=True, ignore_errors=True)
```

Comment:
those paths are in no way enforced, so if someone names their resource differently then this will do nothing